### DISCUSSION TITLE
Cover image for image galleries

### DISCUSSION THREAD

**Discussion by Ezgril** [stashapp/stash#3774] [Ideas]
Hi friends,

It would be interesting to be able to select the cover image of the galleries in the same way that it can be done with the scenes. As an idea for lovers of photo galleries.

Thanks for the work, it´s amazing.

**Comment by DogmaDragon** [comment]
You can name the image `cover.jpg` inside the gallery an it will be used as cover. https://docs.stashapp.cc/in-app-manual/galleries/

Also if you prefer a different word, you can use `gallery_cover_regex` setting to choose a different word. https://docs.stashapp.cc/in-app-manual/configuration/#advanced-configuration-options

**Comment by echo6ix** [comment]
@DogmaDragon Not quite the same though because your `cover.jpg` will get populated in the image gallery when it's scanned. I think OP is requesting the literal parrel to scene cover images where the image can be divorced from the image gallery, where you can manually select an image or paste one in. Why does this have utility? Because people may not want tons of image set cover images populated in their image galleries.

Edit: Let me clarify something, because not everyone may be familiar with image sets. They can have cover images that look like magazine covers, and sometimes they're lower resolution. It's not the type of image you'd care for in your library. It's kind of analogous to requiring movie cover images to be scanned from the library.

**Reply by DogmaDragon** [reply]
I see. It's already a [requested feature by you](https://github.com/stashapp/stash/issues/1432), so giving it a 👍 or making a bounty are the best ways to enourage it's development.

**Reply by echo6ix** [reply]
I've made so many posts I forgot about that. So this is kind of a duplicate of #1432 then.

**Comment by Ezgril** [comment]
Thanks friends, I had no idea.
